pytestqt/_tests/test_basics.py

```diff
@@ -80,6 +80,18 @@ def test_widget_kept_as_weakref(qtbot):
     assert widget() is None
 
 
+def test_header(testdir):
+    res = testdir.runpytest()
+    v = get_versions()
+    res.stdout.fnmatch_lines([
+        '*test session starts*',
+        '{qt_api} {qt_api_version} -- Qt runtime {runtime} -- '
+        'Qt compiled {compiled}'.format(qt_api=v.qt_api,
+                                        qt_api_version=v.qt_api_version,
+                                        runtime=v.runtime, compiled=v.compiled)
+    ])

```

Comment:
I'm a bit worried because the assertion basically re-runs the code under test, so even if there was something weird going on you wouldn't notice.

I don't know if it's possible/easy in conjunction with `pytester`, but maybe I'd be better to monkeypatch the version attributes and then see if the header matches a predefined output text?